scripts/build_changelog.py

```diff
@@ -0,0 +1,68 @@
+import re
+import xml.etree.ElementTree
+from enum import Enum
+from html.parser import HTMLParser
+from pathlib import Path
+from typing import List
+
```

Comment:
what's this file for? 😊

Reply:
Oh, I forgot to remove it.
The scripts generate changelog pages from histores.
https://koxudaxi.github.io/pydantic-pycharm-plugin/changelog/